### PR TITLE
Check for backend port availability before starting

### PR DESCRIPTION
**Pull request by arthasking123**
Update Makefile fix bug #907

### CODE REVIEW

**Comment by rbren** [comment]
This might conflict with https://github.com/OpenDevin/OpenDevin/issues/905

But also might be a good stopgap

**Comment by arthasking123** [comment]
netstat command

**Comment by rbren** [comment]
I think `netstat` has the same problems 🙃 it's not installed by default on many systems.

We already have `nc` in the Makefile, so maybe we just go with that for now

File: evaluation/SWE-bench/Dockerfile [hunk 1]
```diff
@@ -31,7 +31,7 @@ RUN conda env create -f environment.yml
 
 # Add commands
 COPY ./commands.sh .
-RUN source commands.sh
+RUN . commands.sh
```
Comment (rbren) [inline evaluation/SWE-bench/Dockerfile:34]: Any reason for this? I find `source` more readable

Comment (rbren) [reply evaluation/SWE-bench/Dockerfile:34]: I ended up tripping over this in another PR 😄 turns out `source` isn't available on the machine

**Comment by rbren** [comment]
Going to close this one--only active developers will likely be using the makefile going forward, so it should be less of an issue.

Thanks for the proposal though!

**Pull request closed without merging**